cloudplatform/connectivity-oauth/src/main/java/com/sap/cloud/sdk/cloudplatform/connectivity/OAuth2ServiceBindingDestinationLoader.java

```diff
@@ -243,7 +226,7 @@ HttpDestination toProxiedDestination(
             .property(
                 OAuthHeaderProvider.PROPERTY_OAUTH2_RESILIENCE_CONFIG,
                 createTokenRetrievalResilienceConfiguration(name))
-            .buildInternal();
+            .build();
```

Comment:
This change is causing `StackOverfowException`s in our tests.

```suggestion
            .buildInternal();
```